paper.md

```diff
@@ -0,0 +1,88 @@
+
+![Figure 6. The anatomical labels for an sEEG electrode shaft are shown as the contacts progress from depth (starting with 1) to superficial.](figures/Figure_6.png){ width=100% }
+
+![Figure 7. The anatomical surfaces that sEEG electrodes pass through are rendered along with the trajectory of the electrode shaft.](figures/Figure_7.png){ width=100% }
+
+![Figure 8. A time-frequency decomposition of the ECoG data in rendered on top of a 3D image of the brain showing the location of the grid implant.](figures/Figure_8.png){ width=100% }
+
+# Statement of need
+
+Integrating intracranial electrode location and analysis into MNE-Python allows researchers to generate results from raw data in an all-in-one package that has a high development standard, with unit tests and documentation, and includes a wide array of documented functionality for different analyses. Previous work has sometimes been in the form of standalone packages [@HamiltonEtAl2017:2017, @GroppeEtAl2017:2017]. These are difficult to maintain as the group developers and number of users tends to be relatively smaller which often hampers the maintenance of the software as the developers transition to other projects. MNE-Python is a general-purpose electrophysiology analysis package which makes it easier to retain a larger core group of developers and makes it more likely that this package and the intracranial functionality will be maintained and improved upon long-term. Other general purpose packages provide similar functionality [@OostenveldEtAl2011:2011, @TadelEtAl2011:2011] but are in MATLAB and so can be difficult for researchers to integrate into analyses that are scripted in Python. Previous work in Python also provides similar functionality [@HamiltonEtAl2017:2017] but was not designed for sEEG electrode location. Adding the ability to control the size of the electrode contact marker on the slice as well as its opacity and integrating a 3D view makes this task much easier, both for sEEG and ECoG. Additionally, compared to this previous work, MNE-Python supports an SDR morph which is orders of magnitude faster (approximately 15 minutes compared to 15 hours) and comparibly accurate [@GaryfallidisEtAl2014:2014], has a "snap to center" of the electrode contact feature to both increase accuracy and repeatability (this feature uses the center of mass of all voxels that monotonically decrease from the highest intensity voxel nearby the selected location), has an integrated 3D view which updates upon selection and uses a predetermined list of channel names from the recording file to speed up the localization process and eliminate matching errors. In general, the intracranial electrode location and analysis in MNE-Python is implemented at a professional development standards, is feature complete for intracranial analysis and has user-friendly features and is poised to be a well-maintained tool into the future.
```

Comment:
last sentence, suggest change wording and fix typos so it reads: "In general, the intracranial electrode location and analysis in MNE-Python is implemented at a professional development standard, is feature complete for intracranial analysis, has user-friendly features, and is poised to be a well-maintained tool into the future.